Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -760,6 +771,12 @@ public virtual void SetDisabled(bool disabled)
             IsDisabled = disabled;
             Enabled = !disabled;
             SetState(InteractableStates.InteractableStateEnum.Disabled, disabled);
+
+            // Clear the focus if interactable is disabled
+            if(disabled)
```

Comment:
nit: space for if (disabled)

Reply:
Why only clear focus instead of all states?

Reply:
I was only clearing the focus within SetDisable because that was the only state that appeared incorrect within the bugs reported. But I could add a ResetAllStates within SetDisable to clear all the states.